test/files/scalacheck/redblacktree.scala

```diff
@@ -33,7 +33,7 @@ package scala.collection.immutable.redblacktree {
 
     def mkTree(level: Int, parentIsBlack: Boolean = false, label: String = ""): Gen[Tree[String, Int]] =
       if (level == 0) {
-        value(null)
+        const(null)
```

Comment:
oh, it was just deprecated, https://github.com/rickynils/scalacheck/blob/1.11.4/src/main/scala/org/scalacheck/Gen.scala#L290